crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -363,33 +365,36 @@ mod tests {
     #[tokio::test]
     async fn test_avs_writer_methods() {
         let (_container, http_endpoint, _ws_endpoint) = start_anvil_container().await;
-
+        // start_anvil_docker().await;
+        // let http_endpoint = "http://localhost:8545";
```

Comment:
uncomment this line 
and run anvil in a separate terminal 
```
anvil --load-state ./crates/contracts/anvil/dump_state.json --base-fee 0 --gas-price 0
```

and run the test
```
cargo test --package eigen-client-avsregistry --lib --all-features -- writer::tests::test_avs_writer_methods --exact --show-output

```

it will pass , but it fails when doing through testcontainers

Reply:
@pablodeymo